src/Sylius/Bundle/AdminBundle/Resources/config/grids/product_option.yml

```diff
@@ -0,0 +1,28 @@
+sylius_grid:
+    grids:
+        sylius_admin_product_option:
+            driver:
+                name: doctrine/orm
+                options:
+                    class: %sylius.model.product_option.class%
+            fields:
+                code:
+                    type: string
+                    label: sylius.ui.code
+                name:
+                    type: string
+                    label: sylius.ui.name
+                values:
```

Comment:
We do not want to have values in the listing of options. If you add 100 options, things get messy.